graphql_compiler/compiler/filters.py

```diff
@@ -640,6 +678,7 @@ def _get_filter_op_name_and_values(directive):
 COMPARISON_OPERATORS = frozenset({u'=', u'!=', u'>', u'<', u'>=', u'<='})
 PROPERTY_FIELD_OPERATORS = COMPARISON_OPERATORS | frozenset({
     u'between',
+    u'not_in_collection',
```

Comment:
Nit: Let's maintain a consistent relative order between operators here. `not_contains` comes after `contains`, so `not_in_collection` should come after `in_collection`.

Reply:
done!

Reply:
Did this get merged before I changed that?

Reply:
I merged `master` in because this branch was behind and our current setup is such that only branches that are up-to-date can be merged into `master`, to avoid breaking the build. You'll have to pull the change into your local copy of the branch, make the change, commit it, and then push again.

Reply:
just pushed the nit! ready for merge into master.